backend/neo4j-database.md

```diff
@@ -0,0 +1,80 @@
+# Neo4j Database
+
+## Setup
+
+### Indexes
+
+Prepare the database with indexes:
+
+{% tabs %}
+{% tab title="Docker" %}
+If Docker is up, open another terminal and create unique indixes with:
+
+```bash
+# in Human-Connection folder
+$ docker-compose exec neo4j migrate
+```
+{% endtab %}
+
+{% tab title="Local" %}
+If Neo4j database runs, just use the comand:
+
+\(**Attention:** This command is not working at the moment !!! Workaround: Search for file \`backend/neo4j/migrate.sh\` and copy the cypher comands that create the indexes directly into the the cypher shell of Neo4j at [http://localhost:7474/](http://localhost:7474/).\)
```

Comment:
@Tirokk Why is this not working at the moment? It's always better to explain **why** sth. does not work. You can also add this as a hint. The ideal solution is to fix the problem :wink:

Reply:
@Tirokk the script is working for me locally. What's missing is that you have to run it with environment variables:
```bash
NEO4J_USERNAME=neo4j NEO4J_PASSWORD=1234 ./migrate.sh
```
Could you add documentation for it?

Reply:
Hey @roschaefer ,

command on top level running:
```bash
NEO4J_USERNAME=neo4j NEO4J_PASSWORD=1234 ./migrate.sh
```
Gives error `./neo4j/migrate.sh: line 18: cypher-shell: command not found`.
What can I do?

Reply:
It means you don't have `cypher-shell` installed on your machine. There are possible 5 solutions to this problem:

1. Use [Docker](https://www.docker.com/)
2. Alternatively use [Docker](https://www.docker.com/)
3. You could also use [Docker](https://www.docker.com/)
4. But the best solution would probably [Docker](https://www.docker.com/)
5. If you must, you can also setup [Neo4J](https://neo4j.com/) without docker, but you have to install the command line utilities with it.

Reply:
I think @Tirokk is using `Docker`, @roschaefer... I think he is just testing things out locally so that he can update the documentation for someone who does not want to use `Docker` for whatever reasons. 

That being said, I feel like you have made it clear that we are unable to provide support for those who choose to get set up locally and run into issues, although it has been stated by some in the community they run into issues/cannot get `Docker` working for them. At least with the `Docker` option it limits the possible issues one might encounter(?)

As I posted in the Discord channel `Cypher Shell is a command-line tool that is installed as part of the product.`, from their docs https://neo4j.com/docs/operations-manual/current/tools/cypher-shell/, so if @Tirokk, or anyone else, has `Neo4j` locally, they should have `cypher-shell` as a command line tool, although I have the community edition whereas many Mac users have the Desktop version.

In my case, as I prefer not to use `Docker`, and the shell script wasn't working for me, I just copied and pasted them into http://localhost:7474/browser, which others can do, or we can indicate to them that maybe they could give `Docker` a try?

Reply:
Hey,

*the cypher-shell has to be installed separately was the little hint I needed* @roschaefer .
Thanks for that !!!

Sorry, even I have search for this issue, I came not to the clue that the cypher-shell is not installed with the Neo4j Community Edition. One reason I think was, because I had the impression I use it already at [http://localhost:7474/browser](http://localhost:7474/browser).

I heavily use `Docker`, so please don’t make me a foul @roschaefer . 😘🤗

Like @mattwr18 said and you know, I write on the Docs and therefore I have to test the stuff I am writing to see if they work for me.

And I like to run the Webapp cypress tests sometimes, which are not go with Docker as fare as I know, and I don’t like always to switch back to `Docker` if I try out little things, and than switch back to locale.

I’ll try to install the cypher-shell …

Reply:
Yes, you're right. Apologies for being ignorant that you want to update the docs for installation without docker. OK, what we can do: Add a hint that you must have `cypher-shell` installed or copy+paste the statements from the script.